src/App.vue

```diff
@@ -1,28 +1,32 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Blog Project"/>
+    <div id="nav">
+      <router-link to="/">Home</router-link> |
+      <router-link to="/about">About</router-link>
```

Comment:
Change the route and text to "styleguide", this will be a standalone page where we will have all the base components (baseButton, baseInput, baseCard, etc) will be put so it's easy to see and test the UI and functionality for these components